djconnectwise/sync.py

```diff
@@ -501,6 +501,25 @@ def get_page(self, *args, **kwargs):
 
         return records
 
+    def create_new_note(self, target, **kwargs):
+        """
+        Send POST request to ConnectWise to create a new note and then
+        create it in the local database from the response
+        """
+
+        target_data = {}
+
+        if isinstance(target, models.Ticket):
+            target_data['id'] = target.id
+            target_data['type'] = target.record_type
+        else:
+            raise InvalidObjectException("Invalid target type for note \
+            creation: " + str(target.__class__) + ".")
```

Comment:
Please use string formatting ("{}"- style) instead of concatenation. Much easier to read IMO.